docs/conf.py

```diff
@@ -107,11 +107,19 @@ def check_if_nightly(version):
 # The full version, including alpha/beta/rc tags.
 release = version
 
+# If the documentation is not for the nightly, the git clone snippets will show
+# the matching version of the code.
+if version != 'nightly':
+    checkout_branch = '-b {tag}'.format()
```

Comment:
Is the a **v** missing in front of the version? The tags have it: https://github.com/raiden-network/raiden/releases